Windows/debugger.cpp

```diff
@@ -1810,19 +1808,25 @@ DebuggerStatus Debugger::Continue(uint32_t timeout) {
     dbg_last_status = DEBUGGER_TARGET_START;
     return dbg_last_status;
   }
+  if (script != NULL) {
+     HANDLE thread_handle = CreateThread(NULL, 0, (LPTHREAD_START_ROUTINE)system, script, 0, NULL);
+     CloseHandle(thread_handle);
+  }
 
   dbg_last_status = DebugLoop(timeout);
 
   if (dbg_last_status == DEBUGGER_PROCESS_EXIT) {
-    CloseHandle(child_handle);
-    CloseHandle(child_thread_handle);
-    child_handle = NULL;
-    child_thread_handle = NULL;
+    if (!attach_mode) {

```

Comment:
I think these changes need to be reverted (IIUC they were related to the `DEBUGGER_HANGED` change). Basically I think all changes to `debugger.cpp` / `debugger.h` can be reverted at this point.